queries.yaml

```diff
@@ -650,19 +650,42 @@ post_process:
       start_zoom: 8
       end_zoom: 15
 
+  # no lake labels at zoom 0-3:
+  # https://github.com/tilezen/vector-datasource/issues/1730
+  - fn: vectordatasource.transform.drop_properties
```

Comment:
Should this be `drop_properties_with_prefix`?

Reply:
Yup, see 01eeaf1 - we've now got a function specifically for dropping name properties.